lib/galaxy/model/store/__init__.py

```diff
@@ -2753,7 +2753,7 @@ def _finalize(self) -> None:
             out_file = out_file_name[: -len(".zip")]
         else:
             out_file = out_file_name
-        rval = shutil.make_archive(out_file, "zip", self.export_directory)
+        rval = shutil.make_archive(out_file, "fastzip", self.export_directory)
```

Comment:
I think this works only because this files also imports `from galaxy.util.compression_utils import CompressedFile`. Should we re-export `shutil` (maybe as `gx_shutil`) in `lib/galaxy/util/compression_utils.py` and import it in this file from there?

Reply:
> I think this works only because this files also imports `from galaxy.util.compression_utils import CompressedFile`

yes

> Should we re-export `shutil` (maybe as `gx_shutil`) in `lib/galaxy/util/compression_utils.py` and import it in this file from there?

makes sense, sure